src/main/kotlin/org/malachite/estella/services/JobSeekerService.kt

```diff
@@ -0,0 +1,32 @@
+package org.malachite.estella.services
+
+import org.malachite.estella.commons.models.people.JobSeeker
+import org.malachite.estella.people.domain.JobSeekerRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class JobSeekerService(
+    @Autowired private val jobSeekerRepository: JobSeekerRepository,
+    @Autowired private val userService: UserService
+) {
+    fun getJobSeekers(): MutableIterable<JobSeeker> = jobSeekerRepository.findAll()
+
+    fun getJobSeeker(id: Int): JobSeeker = jobSeekerRepository.findByUserId(id).get()
+
+    fun registerJobSeeker(jobSeeker: JobSeeker): JobSeeker =
+        userService.registerUser(jobSeeker.user).let {
+            jobSeeker.copy(user = it)
+        }.let {
+            jobSeekerRepository.save(it)
```

Comment:
Nie wiem czy jak mamy kaskadowość to nie moglibyśmy zrobić po prostu jobSeekerRepository.save

Reply:
Znaczy ja to rżnąłem z HRPartnera chyba, więc jeśli tu tak można, to tam też, tylko o potwierdzenie proszę XD

Reply:
Sprawdziłbym czy się da i jeśli tak to bym zmienił

Reply:
Po przemyśleniu boję się tutaj jeszcze jednej rzeczy - ogólnie co będzie, jeśli dwa razy się zarejestrujemy na ten sam mail. Powinno się wywalić przez usera, ale czy nie zrobi nam to jeszcze jednego job seekera? w sensie czy cascade jest transakcyjny?

Reply:
Ogólnie z logów hibernate'a wychodzi na to, że nie XDD - na pewno próbował zinsertować typa, nie wiem, czy się z tego sam wycofał, zanim rzucił exception, czy jednak nie.

Reply:
Odpowiedź A to powinien być transakcyjny 
Odpowiedź B to sprawdź bazę czy masz jobSeekera z null userem albo z takim co dwa razy pokazuje na to samo ale nie powinno to przejść bo job seeker ma takie samo id jak odpowiadający mu user. Kolejny dowód że był to dobry pomysł @Pinio

Reply:
Znaczy w sumie jak pisałem - nie wiem, co się tam pod spodem odpierdziela. Ogólnie na razie zostawię tak z racji późnej pory, a jako że od jutra zaczynam front, to wolałbym się z tym już nie bawić XDDD

Reply:
Ogólnie sprawa jest prosta - tego nie mogę zapisać za pomocą po prostu save'a, bo w registerUser wysyłamy jeszcze maila z potwierdzeniem - z kolei addJobSeeker jest endpointem dla nas i nie wiem, czy go w ogóle nie wywalić, bo to jest dodanie bez mailowego potwierdzenia. @Rados13 ?

Reply:
Znaczy jak na moje powinieneś zrobić tak jak jest w HrPartnerService. Bo przecież dla każdego rodzaju użytkownika mamy różnego rodzaju mail powitalny, więc nie widzę powodu by mail powitalny dla JobSeekera był w UserService.

Reply:
@Rados13 teraz git powinno być